### ISSUE TITLE
Question: Infinite logger info

### ISSUE THREAD

**Issue by blazekjan** [lucaslorentz/caddy-docker-proxy#298]
Hello, 
sorry to bother you this way, but is there any way how to get rid of this infinite logging info which repeats like every 5 seconds or so?
```
{"level":"info","ts":1633207102.2502303,"logger":"docker-proxy","msg":"Skipping default Caddyfile because no path is set"}
{"level":"info","ts":1633207102.2502704,"logger":"docker-proxy","msg":"Skipping swarm config caddyfiles because swarm is not available"}
{"level":"info","ts":1633207102.2517693,"logger":"docker-proxy","msg":"Skipping swarm services because swarm is not available"}
```
My docker compose:
```
version: "3.7"

services:
  caddy:
    image: lucaslorentz/caddy-docker-proxy:ci-alpine
    container_name: caddy-docker-proxy
    restart: unless-stopped
    ports:
      - 80:80
      - 443:443
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - caddy_data:/data
    networks:
      - caddy

volumes:
  caddy_data: {}

networks:
  caddy:
    external: true
```
Found #102 and also the note:

> Note: This is for Docker Swarm only. Alternativly, use CADDY_DOCKER_CADDYFILE_PATH or -caddyfile-path

I just cant figure out what to do.. set - CADDY_DOCKER_CADDYFILE_PATH=? what path? Create some Caddyfile? I dont use Docker Swarm at all.

Edit: 
Found an issue #289, so I suppose I cant do anything with this.

According to @francislavoie suggestion - it really doesnt work:

```
caddy.log: ""
caddy.log.exclude: docker-proxy
caddy.log: docker-proxy
caddy.log.include: docker-proxy
caddy.log.level: WARN
```
Output:
```
{"level":"info","ts":1633780343.5569098,"logger":"docker-proxy","msg":"Sending configuration to","server":"localhost"}
{"level":"info","ts":1633780343.5604293,"logger":"admin.api","msg":"received request","method":"POST","host":"localhost:2019","uri":"/load","remote_addr":"127.0.0.1:58056","headers":{"Accept-Encoding":["gzip"],"Content-Length": ["736"],"Content-Type":["application/json"],"User-Agent":["Go-http-client/1.1"]}}                                                                                                                                                      {"level":"error","ts":1633780343.5607913,"logger":"admin.api","msg":"request error","error":"loading config: loading new config: setting up custom log 'docker-proxy': include and exclude must not intersect, but found docker-proxy in both lists","status_code":400}
{"level":"error","ts":1633780343.5608947,"logger":"docker-proxy","msg":"Error response from server","server":"localhost","status code":400,"body":"{\"error\":\"loading config: loading new config: setting up custom log 'docker-proxy': include and exclude must not intersect, but found docker-proxy in both lists\"}\n"}
```
I will just leave it emmiting these logs.

**Comment by mikenholt** [comment]
I'm also interested in this because I am not using swarm as well.

**Comment by pn2** [comment]
Maybe I am wrong, but I think this should be changed from a question to a bug.

In my case, `docker info` reports `Swarm: inactive`, yet I get these log entries every 30 seconds, i.e. I end of having nearly 6000 such repetitive lines in the log every day !

```
caddy    |  31-01-2022 18:23:39 [docker-proxy] Skipping swarm services because swarm is not available
caddy    |  31-01-2022 18:24:09 [docker-proxy] Skipping swarm config caddyfiles because swarm is not available
caddy    |  31-01-2022 18:24:09 [docker-proxy] Skipping swarm services because swarm is not available
caddy    |  31-01-2022 18:24:39 [docker-proxy] Skipping swarm config caddyfiles because swarm is not available
caddy    |  31-01-2022 18:24:39 [docker-proxy] Skipping swarm services because swarm is not available
caddy    |  31-01-2022 18:25:09 [docker-proxy] Skipping swarm config caddyfiles because swarm is not available
caddy    |  31-01-2022 18:25:09 [docker-proxy] Skipping swarm services because swarm is not available
```
If this is considered an important information, then I think it would be good if these log entries would only be shown when indeed the Caddyfile changes.

**Comment by jonatasbaldin** [comment]
I worked on #338 to fix this.

**Comment by francislavoie** [comment]
Merged, so I'll close :+1:

**Comment by avirut** [comment]
I'm still receiving `Skipping swarm services because swarm is not available` at 30 seconds intervals on `info` rather than `debug`:
```
{"level":"info","ts":1652032687.586184,"logger":"docker-proxy","msg":"Skipping swarm services because swarm is not available"}
{"level":"info","ts":1652032699.6262467,"logger":"docker-proxy","msg":"Skipping swarm services because swarm is not available"}
{"level":"info","ts":1652032702.183488,"logger":"docker-proxy","msg":"Skipping swarm services because swarm is not available"}
{"level":"info","ts":1652032702.50336,"logger":"docker-proxy","msg":"Skipping swarm services because swarm is not available"}
{"level":"info","ts":1652032732.5104089,"logger":"docker-proxy","msg":"Skipping swarm services because swarm is not available"}
```

Getting the version of `caddy-docker-proxy` with `docker exec -it caddy caddy build-info` yields:
```
github.com/lucaslorentz/caddy-docker-proxy/plugin v0.0.0-20220427185539-819cb0aed5ed h1:Oa9OPp+Hnsltn2nNTHERmw9cHi8degbMl3BtcbrwSJc=
```
The commit above seems to be more recent than the merged PR #338.

Finally, here's my Dockerfile:

```Dockerfile
ARG CADDY_VERSION=2.5.0

FROM caddy:${CADDY_VERSION}-builder AS builder

RUN xcaddy build \
	--with github.com/lucaslorentz/caddy-docker-proxy/plugin \
	--with github.com/caddy-dns/cloudflare \
	--with github.com/greenpau/caddy-security

FROM caddy:${CADDY_VERSION}-alpine

COPY --from=builder /usr/bin/caddy /usr/bin/caddy

CMD ["caddy", "docker-proxy"]
```
The unwanted log seems to originate from: https://github.com/lucaslorentz/caddy-docker-proxy/blob/7bdd39f818fe06254424504156bc32211036382c/plugin/generator/generator.go#L176

I'm happy to try submitting a PR changing `logger.Info` to `logger.Debug` but I'm not super familiar with the whole process so I might not end up following etiquette 100%. Let me know if you'd like me to do so anyways.

**Comment by lucaslorentz** [comment]
Changing it here: https://github.com/lucaslorentz/caddy-docker-proxy/pull/361
